components/script_bindings/domstring.rs

```diff
@@ -415,11 +416,32 @@ impl DOMString {
         self.view().is_empty()
     }
 
-    /// This length (as rust spec) is in bytes if the string would be utf8 not chars.
+    /// The length of this string in UTF-8 code units, each one being one byte in size.
+    ///
+    /// Note: This is different than the number of Unicode characters (or code points). A
+    /// character may require multiple UTF-8 code units.
     pub fn len(&self) -> usize {
         self.view().len()
     }
 
+    /// The length of this string in UTF-8 code units, each one being one byte in size.
+    /// This method is the same as [`DOMString::len`], but the result is wrapped in a
+    /// `Utf8Offset` to be used in code that mixes different kinds of offsets.
```

Comment:
Nit: I think you mean this since that's what the method uses?

```suggestion
    /// `Utf8Length` to be used in code that mixes different kinds of offsets.
```